packages/@aws-cdk/aws-codeguruprofiler/test/integ.profiler-group.ts

```diff
@@ -18,6 +18,14 @@ class ProfilerGroupIntegrationTest extends Stack {
     });
     profilingGroup.grantRead(readAppRole);
 
+    const importedGroup = ProfilingGroup.fromProfilingGroupName(
+      this,
+      'ImportedProfilingGroup',
+      profilingGroup.profilingGroupName,
+    );
+    new CfnOutput(this, 'MyProfilingGroupName', {
```

Comment:
Instead of reusing an existing test, I'd prefer to see a new test with two test cases: one where you explicitly set the profilingGroupName and one where you don't explicitly set it. With the new style of integration tests (see [INTEGRATION_TESTS](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)), you can have multiple test cases and use assertions to test the value. In this case, it's just returning a reference to the Profiling group itself, not to the profilingGroupName. In CloudFormation, this may come out correct, but just reading the raw output with the reference makes it look incorrect as line 6 of the snapshot says 
```
"ProfilingGroupName": "ProfilerGroupIntegrationTestMyProfilingGroup81DA69A3"
``` 
while the output of your test says 
```
 "Outputs": {
  "MyProfilingGroupName": {
   "Value": {
    "Ref": "MyProfilingGroup829F0507"
   }
  }
 },
```

Reply:
See comment below.